src/docTest/groovy/org/gradle/playframework/InDepthUserGuideSamplesIntegrationTest.groovy

```diff
@@ -1,18 +1,23 @@
 package org.gradle.playframework
 
 import org.gradle.exemplar.test.rule.Sample
+import org.gradle.exemplar.test.runner.SamplesRoot
 import org.gradle.playframework.fixtures.archive.ArchiveTestFixture
 import org.gradle.playframework.fixtures.archive.JarTestFixture
 import org.gradle.playframework.fixtures.archive.TarTestFixture
 import org.gradle.playframework.fixtures.archive.ZipTestFixture
+import org.gradle.playframework.runner.GradlePluginSamplesRunner
 import org.gradle.testkit.runner.BuildResult
 import org.gradle.testkit.runner.GradleRunner
 import org.junit.Rule
+import org.junit.runner.RunWith
 import spock.lang.Specification
 
 /**
  * In-depth testing of user guide samples.
  */
+@RunWith(GradlePluginSamplesRunner)
```

Comment:
Not really sure how it was working before without this, but needed to make all docTests run.